app/src/main/java/com/mionix/newsapp/MainActivity.kt

```diff
@@ -36,6 +56,25 @@ class MainActivity : AppCompatActivity() {
         initToolbar()
         initTableLayout()
         initViewPager()
+        initViewModel()
+    }
+
+    private fun initViewModel() {
+        this@MainActivity.let {
```

Comment:
Tại sao phải dùng thế này?
mActivityViewModel = ViewModelProviders.of(this).get(ActivityViewModel::class.java) <- Như thế này không được à?